examples/braket_features/Getting_Started_with_OpenQASM_on_Braket.ipynb

```diff
@@ -251,7 +251,7 @@
   },
```

Comment:
<p>We can check whether a device supports partial measurement by inspecting the <code>requiresAllQubitsMeasurement</code> field in its action properties; if it is <code>False</code>, then partial measurement is supported.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/540/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Reply:
Updated this text.